packages/cli/src/eventbus/eventBus.controller.ts

```diff
@@ -213,52 +242,52 @@ eventBusRouter.post(
 			}
 			if (result) {
 				await result.saveToDb();
-				return {
-					...result,
-					eventBusInstance: undefined,
-				};
+				return res.send({
```

Comment:
all instances of `return res.send({ data: xyz })` can be replaced with `return xyz`.